net-vpn/networkmanager-fortisslvpn/networkmanager-fortisslvpn-1.3.90.ebuild

```
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GNOME_ORG_MODULE="NetworkManager-${PN##*-}"
+GNOME2_LA_PUNT="yes"
+GNOME2_EAUTORECONF="yes"
+
+inherit gnome2
+
+DESCRIPTION="NetworkManager Fortinet SSLVPN compatible plugin"
+HOMEPAGE="https://wiki.gnome.org/Projects/NetworkManager"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gtk"
+
+RDEPEND="
+	>=net-misc/networkmanager-1.2:=
+	>=dev-libs/glib-2.32:2
+	net-dialup/ppp:=
+	>=net-vpn/openfortivpn-1.2.0
+	gtk? (
+		>=app-crypt/libsecret-0.18
+		>=net-libs/libnma-1.2.0
+		>=x11-libs/gtk+-3.4:3
+	)
+"
+DEPEND="${RDEPEND}
+	dev-util/gdbus-codegen
+	>=sys-devel/gettext-0.19
+	virtual/pkgconfig
+"
```

These all look to be `BDEPEND` instead.
```suggestion
DEPEND="${RDEPEND}"
BDEPEND="dev-util/gdbus-codegen
	>=sys-devel/gettext-0.19
	virtual/pkgconfig"